src/enkf/gridio_fv3reg.f90

```diff
@@ -1064,6 +1152,58 @@ subroutine writegriddata_pnc(vars3d,vars2d,n3d,n2d,levels,ndim,grdin,no_inflate_
     real(r_single), dimension(npts,ndim,nbackgrounds,1), intent(inout) :: grdin
     logical, intent(in) :: no_inflate_flag
   end subroutine writegriddata_pnc
+  subroutine type_bound_setupfilebank(this,fileid1,fv3fn1,fileid2,fv3fn2)
+       class (type_fv3lamfilebank) :: this  
+       integer(i_kind) fileid1
+       integer(i_kind), optional :: fileid2
+       character(len=*)::fv3fn1
+       character(len=*),optional ::fv3fn2
+       if (present (fileid2)) then
+         this%l_filecombined=.false. 
+         this%fv3lamfilename(1)=trim(fv3fn1)
+         this%fv3lamfilename(2)=trim(fv3fn2)
+         this%fv3lam_fileid(1)=fileid1
+         this%fv3lam_fileid(2)=fileid2
+       else
+         this%l_filecombined=.true. 
+         this%fv3lamfilename(1)=fv3fn1
+         this%fv3lam_fileid(1)=fileid1
+       endif
+   end subroutine type_bound_setupfilebank
+       subroutine type_bound_getidfn(this,vnamloc,fileid,fv3fn)
+       class (type_fv3lamfilebank) :: this  
+       integer(i_kind) fileid
+       character(len=*)::fv3fn,vnamloc
+       if (.not.this%l_filecombined) then
+         if(ifindstrloc(vardynvars,vnamloc)> 0)  then  
+           fv3fn=trim(this%fv3lamfilename(1))
+           fileid=this%fv3lam_fileid(1)
+         else if(ifindstrloc(vartracers,vnamloc)> 0)  then  
+           fv3fn=trim(this%fv3lamfilename(2))
+           fileid=this%fv3lam_fileid(2)
+         else
+           write(6,*)"the varname ",trim(vnamloc)," is not recognized in the ype_bound_getidfn, stop"
```

Comment:
ype_bound_getidfn   --> type_bound_getidfn

Reply:
OK, then, I changed fv3lamfilebank to fv3lamfile.